x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.test.js

```diff
@@ -344,7 +359,7 @@ describe('get mapbox color expression', () => {
         const colorProperty = makeProperty(dynamicStyleOptions);
         expect(colorProperty._getMbColor()).toEqual([
           'step',
-          ['coalesce', ['feature-state', '__kbn__dynamic__myField__lineColor'], 9],
+          ['coalesce', ['feature-state', '__kbn__dynamic__foobar__lineColor'], 9],
```

Comment:
Shouldn't the feature state name just be the property name? Why are we still adding `__kbn__dynamic__` prefix and `__lineColor` suffix? Those concepts are no longer needed since the values are not getting scaled and we are no longer mutating the original property values

Reply:
test needs to be updated still